public/media_engagement/styles.css

```diff
@@ -1,12 +1,25 @@
-.nav {
-    background: #ffff;
-    width: 100%;
-    max-width: initial;
-    height: 60px;
+.logo {
+    height: 3rem;
+}
+.navbar {
+    box-shadow: 0 2px 2px -2px steelblue;
+}
+.navbar-brown{
+    color: #8d3538;
 }
 .nav-link {
     color: #8d3538;
 }
+.nav-link:hover {
+    color: steelblue;
+}
+.active-link {
+    border-bottom: 2px brown solid;
+}
+.active-item {
+    color: #8d3538;
+    background-color: lightcyan;
```

Comment:
Can you pick a softer, more AVF-like colour. This is a bit ugly

Reply:
Okay...

Reply:
Actually I wonder if the color property is enough, and we don't need a background here?

Reply:
changed